plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/schema/confluent/KafkaWithConfluentSchemaRegistryQueryRunner.java

```diff
@@ -15,6 +15,7 @@
 
```

Comment:
Can you please explain in commit message what was flaky and why?

Reply:
Will do - and I'll make sure to find out why:) It seems like a suppressed or wrapped error from KafkaAvroSerializer but will verify.